documentation/modules/auxiliary/gather/snare_registry.md

```diff
@@ -0,0 +1,105 @@
+## Description
+
+  This module uses the Registry Dump feature of the [Snare Lite for Windows](https://sourceforge.net/projects/snare/) HTTP service on 6161/TCP to retrieve the Windows registry. The Dump Registry functionality is unavailable in Snare Enterprise.
+
+  Note: The Dump Registry functionality accepts only one connected client at a time. Requesting a large key/hive will cause the service to become unresponsive until the server completes the request.
+
+
+## Vulnerable Application
+
+  SNARE (System iNtrusion Analysis and Reporting Environment) is a series of log collection agents that facilitate centralised analysis of audit log data.
+
+  This module has been tested successfully with Snare Lite for Windows version 4.0.2.0 on Windows XP SP3.
+
+  Installers for Snare Lite are no longer available and the product is unsupported.
```

Comment:
I had a look for the software on Web Archive and Sourceforge. It's gone. However, the free trial is available here:

* https://drive.google.com/open?id=0B17PaDdG2x0BXzI3RzBLNFowSkU

I'll update the docs.